components/mdc/Fab/Fab.svelte

```diff
@@ -24,7 +24,7 @@ onMount(() => {
 }
 </style>
 
-<a class="mdc-fab" aria-label={label} bind:this={element} href={url}>
+<a class="mdc-fab {$$props.class}" aria-label={label} bind:this={element} href={url}>
```

Comment:
does this need to be mentioned in CHANGELOG?

Reply:
Yes, I've added it in the next PR.